mivs/models.py

```diff
@@ -107,8 +107,8 @@ class IndieStudio(MagModel):
     staff_notes = Column(UnicodeText, admin_only=True)
     registered  = Column(UTCDateTime, server_default=utcnow())
 
-    games = relationship('IndieGame', backref='studio')
-    developers = relationship('IndieDeveloper', backref='studio')
+    games = relationship('IndieGame', backref='studio', order_by='IndieGame.title')
```

Comment:
While testing the new changes with multiple games, I noticed the page would load the list in a different order randomly. This was crazy annoying, so I looked up a fix. Since I was doing that, I figured I'd make `developers` an ordered relationship as well.